operations/mimir-mixin/docs/playbooks.md

```diff
@@ -1011,6 +1011,99 @@ How to **investigate**:
   1. The alert fired because of a bug in Mimir: fix it.
   1. The alert fired because of a bug or edge case in the continuous test tool, causing a false positive: fix it.
 
+## Codified errors
+
+Mimir has some codified error IDs that you might see in HTTP responses or logs.
+These error IDs allow you to quickly look up more details in the documentation below.
+
+### err-mimir-missing-metric-name
+
+This non-critical error occurs when Mimir receives a write request containing a series without metric name.
+The metric name is required for each series.
+
+> **Note**: Invalid series are skipped during the ingestion, while other valid series within the same request are ingested.
+
+### err-mimir-metric-name-invalid
+
+This non-critical error occurs when Mimir receives a write request containing a series with an invalid metric name.
+A metric name can only contain a well known set of characters, as defined by [Prometheus specifications](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels).
+Specifically, it may only contain ASCII letters, digits, underscores and colons.
+
+> **Note**: Invalid series are skipped during the ingestion, while other valid series within the same request are ingested.
+
+### err-mimir-max-label-names-per-series
+
+This non-critical error occurs when Mimir receives a write request containing a series with a number of labels exceeding the configured limit.
+The limit is used to protect the system stability from potential abuses or mistakes, and it can be configured on a per-tenant basis using `-validation.max-label-names-per-series`.
+
+> **Note**: Invalid series are skipped during the ingestion, while other valid series within the same request are ingested.
+
+### err-mimir-label-invalid
```

Comment:
I'd make this `err-mimir-label-name-invalid`, to be consistent with the error, which fails when label name is invalid.

Reply:
Error IDs were kept to match existing `reason` label values used in metrics, to have a 1:1 matching.